src/styles/blocks/header.scss

```
@@ -0,0 +1,70 @@
+.header {
+  background-image: url(../images/header.jpg);
+  background-size: cover;
+  background-position: center;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    box-sizing: border-box;
+    padding: $headerTopPadding 0 80px;
+    height: 100vh;
+  }
+
+  &__call {
+    text-decoration: none;
+    position: relative;
+
+    @include onDesktop {
+      &::before {
+        display: block;
+        position: absolute;
+        top: 50%;
+        transform: translate(-100%, -50%);
```

```suggestion
        transform: translate(calc(-100% - 20px), -50%);
```
лучше отступ задавать вот так, вместо padding-right